MYREADME.md

```diff
@@ -47,7 +47,7 @@ I would like a default airport capacity that can be overridden as appropriate
 3. Need to change the default capacity of the airport by a given number of planes
 
 ### Tests For User Story 2
-4. The maximum length of `airportPlanes` is output as `defaultCapacity`
+4. The maximum length of `airportPlanes` is output as `defaultCapacity` |DONE|
 5. The `defaultCapacity` of the airport increases when `increaseCapacity()` is called |DONE|
 6. The `defaultCapacity` of the airport decreases when `decreaseCapacity()` is called |DONE|
```

Comment:
Should just be 1 function to increase or decrease to reduce the amount of code. Changed in a later commit.